### ISSUE TITLE
ganeti-confd needs to be better at reporting errors

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#461]
Originally reported of Google Code with ID 405.

```
Currently if e.g. the job queue can't be opened, queries succeed with just an empty result. I believe the error should be propagated properly. Same if the config can't be opened.
```
Originally added on 2013-04-03 11:49:58 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-03 12:45:36 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit be0cb2d73124f93be0a7de7ac9079430696252a9
Author: Michele Tartara <mtartara@google.com>
Date:   Thu Apr 4 18:49:49 2013 +0200

    Properly export errors while reading job list
    
    In case of problems while reading the job list from disk (such as permission
    errors) confd would silently fail, writing a warning on the log file but
    sending an empty list and no error message to the client.
    
    Also, tests have been updated in accordance to the new interface of the modified
    functions.
    
    This commit fixes this problem [Issue 405].
    
    Signed-off-by: Michele Tartara <mtartara@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2013-04-08 07:52:20 +0000 UTC.

Mentioned Issue with GoogleCode ID 405 has been migrated to GitHub with ID #461.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
